testing/check_postgres/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Contributor: G.J.R. Timmer <gjr.timmer@gmail.com>
+# Maintainer: G.J.R. Timmer <gjr.timmer@gmail.com>
+pkgname=check_postgres
+pkgver=2.22.0
+pkgrel=0
+pkgdesc="PostgreSQL monitoring script"
+url="https://bucardo.org/wiki/Check_postgres"
+arch="noarch"
+license="BSD"
+depends="postgresql perl"
+makedepends="perl-dev"
+install=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/bucardo/check_postgres/archive/${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	default_prepare || return 1
+}
```

Comment:
This is not needed, just omit whole `prepare()` function and default_prepare will be used by default.